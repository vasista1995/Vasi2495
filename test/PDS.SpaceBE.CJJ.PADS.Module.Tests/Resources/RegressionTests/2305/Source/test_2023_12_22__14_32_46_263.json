{
  "IdSource": "SPACEACT2:CJJ:BE:2Z34905I:BE1KOR:Test1:CO:58471:243031367:0",
  "Data_flat_Metadata": {
    "PackageFamily": "MDIP",
    "Package": "PG-MDIP-24-12",
    "ParameterClass": "NON_KEY",
    "Device": "K9051E59531",
    "SpecName": "Test1",
    "SalesName": "IKCM15F60GA",
    "DeviceFamily": "K9051",
    "PackageGroup": "MDIP-FP-24",
    "PackageClass": "MDIP-FP",
    "BusinessSegment": "59",
    "MoveInQuantity": "895",
    "MoveOutQuantity": "892",
    "ProcessOwner": "PTE",
    "BeSegmentName": "CHE",
    "ManufacturingWipLevel": "ASSY",
    "Baunumber": "95076911",
    "OperatorId": "CHEMFOPS1",
    "Lot": "2Z34905I",
    "ParameterName": "CO",
    "ParameterUnit": "%",
    "ParameterOper": "Test1",
    "ParameterFacility": "BE1KOR",
    "SiteKey": "CJJ",
    "ChannelId": "58471",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-12-10T13:52:34Z",
    "CreatedTimestamp": "2023-12-10T22:52:34Z",
    "UpdatedTimestamp": "2023-12-10T22:52:34Z",
    "UpdatedTimestampUtc": "2023-12-10T13:52:34Z",
    "ExportedTimestampUtc": "2023-12-10T22:52:33Z",
    "ExportedTimestamp": "2023-12-10T22:52:33Z",
    "SampleTimestampUtc": "2023-12-10T13:52:33Z",
    "SampleTimestamp": "2023-12-10T22:52:33Z",
    "SourceDataLevel": "L",
    "BasicType": "K9051E",
    "Equipment": "LTE007",
    "ChannelName": "CO\\TEST1\\MDIP_FP\\All",
    "ChannelDescr": "CO\\TEST1\\MDIP_FP\\All",
    "RvStoreFlag": "N",
    "CalcCreated": "03/11/2021 08:13:23",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 452942,
    "ViolCount": 0,
    "Owner": "PROD",
    "LdsID": "2305",
    "IsCalcParam": 1,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "N",
        "SpecHigh": 100.0,
        "SpecTarget": 0.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "NNYYYNNYYNYYYYYYYYYNNNYYYYYYYNNNNYYYY"
      },
      "SpaceAggregates": {
        "Mean": "0",
        "Median": "0",
        "Sigma": "0",
        "Min": "0",
        "Max": "0"
      },
      "ControlLimits": {
        "CtrlMeanLowEnabled": "N",
        "CtrlMeanTargetEnabled": "N",
        "CtrlMeanHighEnabled": "N",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 0.0,
        "ExtEwmaMV": 0.0,
        "ExtMsMV": 0.0,
        "EwmaS": 0.0,
        "EwmaR": 0.0,
        "ExtEwmaMVLCL": 0.0,
        "ExtEwmaMVCenter": 0.0,
        "ExtEwmaMVUCL": 0.0,
        "ExtMaMVLCL": 0.0,
        "ExtMaMVCenter": 0.0,
        "ExtMaMVUCL": 0.0,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.0,
        "ExtMSMVCenter": 0.0,
        "ExtMSMVUCL": 0.0,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.0,
        "RawUal": 0.0,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 243031367,
        "Seqnr": 0,
        "Value": 0.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 1
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CJJ:BE:BE1KOR:Test1:CO:58471",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:2Z34905I",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CJJ:BE:2Z34905I:BE1KOR:Test1:CO:58471:243031367:0"
  }
}