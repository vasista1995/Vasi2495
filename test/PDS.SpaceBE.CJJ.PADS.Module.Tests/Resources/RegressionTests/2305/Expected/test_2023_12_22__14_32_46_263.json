{
  "Data_flat_Metadata": {
    "Baunumber": "95076911",
    "OperatorId": "CHEMFOPS1",
    "MeasLot": "2Z34905I",
    "ParameterOper": "Test1",
    "ParameterFacility": "BE1KOR",
    "SiteKey": "CJJ",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-12-10T13:52:33Z",
    "BeginTimestamp": "2023-12-10T22:52:33Z",
    "EndTimestampUtc": "2023-12-10T13:52:33Z",
    "EndTimestamp": "2023-12-10T22:52:33Z",
    "Equipment": "LTE007"
  },
  "Data_flat_StrangeMetadata": {
    "PackageFamily": "MDIP",
    "Package": "PG-MDIP-24-12",
    "Device": "K9051E59531",
    "SpecName": "Test1",
    "SalesName": "IKCM15F60GA",
    "DeviceFamily": "K9051",
    "PackageGroup": "MDIP-FP-24",
    "PackageClass": "MDIP-FP",
    "BusinessSegment": "59",
    "MoveInQuantity": "895",
    "MoveOutQuantity": "892",
    "ProcessOwner": "PTE",
    "BeSegmentName": "CHE",
    "ManufacturingWipLevel": "ASSY",
    "BasicType": "K9051E"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 0,
          "SampleId": 243031367,
          "Value": "NaN",
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-10T22:52:33Z",
          "SampleTimestampUtc": "2023-12-10T13:52:33Z",
          "CreatedTimestamp": "2023-12-10T22:52:34Z",
          "CreatedTimestampUtc": "2023-12-10T13:52:34Z",
          "UpdatedTimestamp": "2023-12-10T22:52:34Z",
          "UpdatedTimestampUtc": "2023-12-10T13:52:34Z",
          "NumViolations": 0,
          "SampleMean": 0.0,
          "SampleMin": 0.0,
          "SampleMax": 0.0,
          "SampleMedian": 0.0,
          "SampleStdev": 0.0,
          "SampleQ1": "NaN",
          "SampleQ3": "NaN",
          "SampleSize": 1
        }
      ],
      "ParameterClass": "NON_KEY",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "N",
          "spec_high": 100.0,
          "target": 0.0
        },
        "ControlLimits": {
          "CtrlMeanLowEnabled": "N",
          "CtrlMeanTargetEnabled": "N",
          "CtrlMeanHighEnabled": "N",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "N",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "CtrlRangeHighEnabled": "N"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 1,
        "base_count": 1,
        "flagged_count": 0,
        "mean": 0.0,
        "min": 0.0,
        "max": 0.0,
        "NumViolations": 0,
        "ViolationComments": "",
        "PrimaryViolation": "",
        "PrimaryViolationComments": "",
        "Samples": "243031367"
      },
      "ParameterName": "CO",
      "ParameterUnit": "%",
      "ChannelId": "58471",
      "SourceDataLevel": "L",
      "channel_name": "CO TEST1 MDIP_FP All",
      "channel_description": "CO TEST1 MDIP_FP All",
      "RvStoreFlag": "N",
      "created_timestamp_utc": "2023-12-10T13:52:34Z",
      "created_timestamp": "2023-12-10T22:52:34Z",
      "updated_timestamp_utc": "2023-12-10T13:52:34Z",
      "updated_timestamp": "2023-12-10T22:52:34Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-12-10T13:52:34Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:CJJ:BE:BE1KOR:Test1:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:2Z34905I",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "CJJ:BE:2Z34905I:BE1KOR:Test1:CO:58471:243031367:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-12-10T22:52:33Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot:2Z34905I:SPACEAGGED2:CJJ:BE:BE1KOR:Test1::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "CJJ"
  }
}